action.yml

```diff
@@ -144,7 +148,7 @@ runs:
         ACME_SH_TEMP_DIR="$(mktemp -d)"
         ACME_SH_TEMP_FILE_FULLCHAIN="$ACME_SH_TEMP_DIR/fullchain.pem"
         ACME_SH_TEMP_FILE_KEY="$ACME_SH_TEMP_DIR/key.pem"
-        ~/.acme.sh/acme.sh --install-cert -d "$ACME_SH_FIRST_DOMAIN" --fullchain-file "$ACME_SH_TEMP_FILE_FULLCHAIN" --key-file "$ACME_SH_TEMP_FILE_KEY"
+        ~/.acme.sh/acme.sh --install-cert -d "$ACME_SH_FIRST_DOMAIN" --fullchain-file "$ACME_SH_TEMP_FILE_FULLCHAIN" --key-file "$ACME_SH_TEMP_FILE_KEY" "$ACME_SH_INPUT_INSTALL_ARGS_PREPENDED"
```

Comment:
It shouldn't be wrapped with double quotes, or multiple arguments splitted with spaces will be treat as one argument.

Reply:
Additionally, since it's in the end of the argument list, it should be named "appended" instead of "prepended".